### PR TITLE
fix NPE in linkd tests

### PR DESCRIPTION
**Pull request by RangerRick**
This PR fixes some NPEs we saw in CircleCI tests (example [here](https://app.circleci.com/pipelines/github/OpenNMS/opennms/9359/workflows/f706b6af-27ae-41f3-83a1-34e9f26bf620/jobs/51879))

### CODE REVIEW

**Comment by RangerRick** [comment]
I honestly don't understand how this ever worked.

File: opennms-services/src/test/java/org/opennms/netmgt/nb/Nms0001NetworkBuilder.java [hunk 1]
```diff
@@ -52,10 +52,10 @@
     FROH_IP_IF_MAP.put(InetAddressUtils.addr("10.0.1.2"), 606);
     FROH_IP_IF_MAP.put(InetAddressUtils.addr("66.3.9.1"), 613);
     FROH_IP_IF_MAP.put(InetAddressUtils.addr("10.246.16.245"), 608);
-    FROH_IP_IF_MAP.put(InetAddressUtils.addr("66-3-123-1.la-04.cvx.algx.net"), 668);
```
Comment (cgorantla) [inline opennms-services/src/test/java/org/opennms/netmgt/nb/Nms0001NetworkBuilder.java:55]: Is it just that these IPs are no longer accessible and throwing exception in static block that would make all other elements to be not added ?

**Review by cgorantla — APPROVED** [review]
LGTM

File: opennms-services/src/test/java/org/opennms/netmgt/nb/Nms0001NetworkBuilder.java (hunk 1 above)
Comment (RangerRick) [reply opennms-services/src/test/java/org/opennms/netmgt/nb/Nms0001NetworkBuilder.java:55]: Yeah, they're not IPs at all, and don't parse.  It honestly makes no sense to me why this wasn't failing for all of time. The _entire_ block of map-filling doesn't happen.

**Comment by rssntn67** [comment]
I wrote some notes on documentation.

**Pull request merged**